wallet-new/integration/Functions.hs

```diff
@@ -38,8 +38,8 @@ import           Cardano.Wallet.Client (ClientError (..), Response (..), Servant
 import           Pos.Core (getCoin, mkCoin, unsafeAddCoin, unsafeSubCoin)
 import qualified Pos.Wallet.Web.ClientTypes.Types as V0
 
-import           Error
-import           Types
+import           Error ( WalletTestError(..), showConstr )
+import           Types (WalletState, ActionProbabilities, WalletTestMode, wallets, getWeight, lastAction, successActions, transactions, addresses, walletsPass, accounts, nonGenesisWallets, actionsNum, Action (..))
```

Comment:
For internal modules that we use heavily, adding explicit imports only adds noise IMO. This is also way over the 80 char column limit